app/src/components/shared/interfaces/surveyForm.interfaces.ts

```diff
@@ -0,0 +1,22 @@
+export interface course {

```

Comment:
I also have some interfaces built up for this stuff but I suspect your PR will be approved and merged first so let's chat once Hexuan's PR is ready and we'll sort out our interfaces. Nice work!